src/model_config_utils.h

```diff
@@ -86,6 +86,14 @@ Status GetNormalizedModelConfig(
     const std::string& model_name, const std::string& path,
     const double min_compute_capability, inference::ModelConfig* config);
 
+/// Auto-complete the instance count based on instance kind and backend name.
+/// For KIND_CPU the default instance count is 2 except for the pytorch backend.
+/// \param group The instance group to set the count for.
+/// \param backned The backend name to check against.
+/// \return The error status.
+Status SetDefaultInstanceCount(
+    ::inference::ModelInstanceGroup* group, const std::string& backend);
```

Comment:
any specific reason for the leading `::` before `::inference` ? This isn't done in the rest of the file, so just curious.

Reply:
I saw it being done in other files so I thought this was the norm. Fixed.